src/main/java/com/zaxxer/hikari/HikariConfig.java

```diff
@@ -243,8 +243,8 @@ public long getValidationTimeout()
    @Override
    public void setValidationTimeout(long validationTimeoutMs)
    {
-      if (validationTimeoutMs < 250) {
-         throw new IllegalArgumentException("validationTimeout cannot be less than 250ms");
+      if (validationTimeoutMs < 1000) {
+         throw new IllegalArgumentException("validationTimeout cannot be less than 1000ms");
       }
```

Comment:
You can't do this, this was specifically changed for a reason:

https://github.com/brettwooldridge/HikariCP/issues/547

Please check the "Blame" history of a file before changing particular lines.  The blame for this line would show this bug specifically.